committee-steering/README.md

```diff
@@ -18,16 +18,11 @@ The [charter](https://git.k8s.io/steering/charter.md) defines the scope and gove
 
 ## Members
 
-* Brian Grant (**[@bgrant0607](https://github.com/bgrant0607)**), Google
-* Brendan Burns (**[@brendandburns](https://github.com/brendandburns)**), Microsoft
+* Christoph Blecker (**[@cblecker](https://github.com/cblecker)**), Red Hat
```

Comment:
FWIW, I think it's worth calling out Emeritus members as well.

Reply:
We hadn't previously done this in sigs.yaml for steering before (Tim and Quinton aren't there), but had in git.k8s.io/steering/README.md. Happy to add them here too if we'd like

Reply:
The readme is sufficient, I've long since lost track of all the redundant places we've scrawled things down...